src/Functional.CQS.AOP.IoC.PureDI.MetricsCapturing.Tests/Functional.CQS.AOP.IoC.PureDI.MetricsCapturing.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Consider enabling `TreatAllWarningsAsErrors` for all projects, in both `Debug` and `Release` configurations.